I.write-tests/01-greet-people/greet-people.test.js

```diff
@@ -1,7 +1,15 @@
 test('print list of names prefixed with Hello', function () {
     // Arrange
+    // Given
+
+var greetPeople = require('./greet-people');
```

Comment:
This isn't wrong, but normally we put the `require` at the top of the file, separated from the rest of your code